build.gradle.kts

```diff
@@ -13,22 +13,22 @@ kotlinDslPluginOptions {
 
 repositories {
     mavenCentral()
-    gradlePluginPortal()
+    maven {
+      setUrl("https://artifacts.openmicroscopy.org/artifactory/maven")
+    }
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
If I understand correctly, this change will force Java 11 as the minimum version for building the `omero-*` components (which are all using this plugin).

By itself, this is completely acceptable but this raises the question of whether Java 11 would also become the minimum runtime requirement for these components. By default, this would be the case with this change.

If we wanted to have Java 11 as the minimum build requirement but keep Java 8 as the minimum runtime requirement, it should be possible to use the `release` flag available in Java 10+ https://docs.gradle.org/current/userguide/building_java_projects.html#sec:compiling_with_release

Reply:
You are correct. Java version is one of the point of discussion.
I already had to do it for insight but this is obviously less intrusive